packages/edit-navigation/src/components/layout/index.js

```diff
@@ -66,6 +66,7 @@ export default function Layout( { blockEditorSettings } ) {
 								templateLock: 'all',
 								hasFixedToolbar: true,
 							} }
+							useSubRegistry={ false }
```

Comment:
This is needed, as without this `wp.data.select( 'core/block-editor' ).getBlocks()` isn't usable—but it's needed in the test to assert which blocks are present.

Reply:
Just wanted to whine here. These true / false flags are so hard to pin down on meaning when reading code.  Looking [here](https://github.com/WordPress/gutenberg/blob/master/packages/block-editor/src/components/provider/README.md) it's not even mentioned.